compel/arch/aarch64/src/lib/infect.c

```diff
@@ -60,7 +60,7 @@ int sigreturn_prep_fpu_frame_plain(struct rt_sigframe *sigframe, struct rt_sigfr
 }
 
 int compel_get_task_regs(pid_t pid, user_regs_struct_t *regs, user_fpregs_struct_t *fpsimd, save_regs_t save,
-			 void *arg, __maybe_unused unsigned long flags)
+			 void *arg, __maybe_unused unsigned long flags, __maybe_unused bool pre_dump)
```

Comment:
Since this argument is now used with both `criu pre-dump` and `criu dump --leave-running`, it might be better to use more generic name than `pre_dump`.